about.html

```diff
@@ -12,53 +12,87 @@
 </head>
 
 <body>
+  <div class="bg-div">
+    <img class="logo-img" src="img/lambda-black.png">
```

Comment:
Since you are using flexbox, it is a lot easier to layout the `<a>` elements without the `<ul><li>` elements.  There is quite a bit of CSS you have to get rid of the bullet points, laid out properly, margin fixes.  See my comments in your CSS file.